eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.7.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
+    <VersionPrefix>17.7.1</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
I'm tempted to merge everything we want to take for 17.7 into a single .1 version bump, but that will be a bit of a hassle of manual work. We can also just follow the regular procedure. Preferences?